From 31fd98f1b6d3ce42f45cbc5c7c4ea3aea734b5c3 Mon Sep 17 00:00:00 2001
From: csev1755 <49533775+csev1755@users.noreply.github.com>
Date: Sun, 23 Nov 2025 07:53:00 -0600
Subject: [PATCH 05/11] add diffconfig

---
 diffconfig | 32 ++++++++++++++++++++++++++++++++
 1 file changed, 32 insertions(+)
 create mode 100644 diffconfig

diff --git a/diffconfig b/diffconfig
new file mode 100644
index 00000000000..646c1cc2ee7
--- /dev/null
+++ b/diffconfig
@@ -0,0 +1,32 @@
+CONFIG_TARGET_ramips=y
+CONFIG_TARGET_ramips_rt305x=y
+CONFIG_TARGET_ramips_rt305x_DEVICE_wansview_ncs601w=y
+CONFIG_PACKAGE_alsa-lib=y
+CONFIG_PACKAGE_glib2=y
+CONFIG_PACKAGE_gst1-mod-alaw=y
+CONFIG_PACKAGE_gst1-mod-alsa=y
+CONFIG_PACKAGE_gst1-mod-mulaw=y
+CONFIG_PACKAGE_gst1-mod-rtp=y
+CONFIG_PACKAGE_gst1-mod-udp=y
+CONFIG_PACKAGE_gstreamer1-libs=y
+CONFIG_PACKAGE_gstreamer1-plugins-base=y
+CONFIG_PACKAGE_gstreamer1-plugins-good=y
+CONFIG_PACKAGE_gstreamer1-utils=y
+CONFIG_PACKAGE_libatomic=y
+CONFIG_PACKAGE_libattr=y
+CONFIG_PACKAGE_libffi=y
+CONFIG_PACKAGE_libgst1audio=y
+CONFIG_PACKAGE_libgst1net=y
+CONFIG_PACKAGE_libgst1pbutils=y
+CONFIG_PACKAGE_libgst1rtp=y
+CONFIG_PACKAGE_libgst1tag=y
+CONFIG_PACKAGE_libgstreamer1=y
+CONFIG_PACKAGE_libjpeg-turbo=y
+CONFIG_PACKAGE_libpcre2=y
+CONFIG_PACKAGE_libpthread=y
+CONFIG_PACKAGE_librt=y
+CONFIG_PACKAGE_libxml2=y
+CONFIG_PACKAGE_mjpg-streamer=y
+CONFIG_PACKAGE_mjpg-streamer-input-uvc=y
+CONFIG_PACKAGE_mjpg-streamer-output-http=y
+# CONFIG_PACKAGE_libgst1video is not set
-- 
2.43.0

